# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/vtk/hotfix-hdf5-parallel.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- VTK-9.7.0/ThirdParty/hdf5/CMakeLists.txt.vanilla	2026-09-27 20:15:05.080629701 +0300
+++ VTK-9.7.0/ThirdParty/hdf5/CMakeLists.txt	2026-09-27 20:16:59.422568532 +0300
@@ -33,7 +33,7 @@
     COMPONENTS    C HL
     TARGETS       hdf5::hdf5
                   hdf5::hdf5_hl
-    USE_VARIABLES HDF5_IS_PARALLEL
+    USE_VARIABLES HDF5_PROVIDES_PARALLEL
     STANDARD_INCLUDE_DIRS)
 
 set(vtkhdf5_is_parallel 0)
